core/modules/wdb/QueryRunner.h

```diff
@@ -148,7 +148,7 @@ class QueryRunner : public wbase::TaskQueryRunner, public std::enable_shared_fro
     std::atomic<bool> _cancelled{false};
     std::atomic<bool> _removedFromThreadPool{false};
     mysql::MySqlConfig const _mySqlConfig;
-    std::unique_ptr<mysql::MySqlConnection> _mysqlConn;
+    std::unique_ptr<mysql::MySqlConnection> _mysqlConn{nullptr};
```

Comment:
That is not needed because the `unique_ptr` is initialized with `nullptr`.

Reply:
There was a crash where gdb said the destructor was called for _mysqlConn when it should have been nullptr. The only ways I can see that happening are:
1) memory corruption
2) Somehow the scheduler scheduled this task to run twice. (log messages)
3) It wasn't initialized to nullptr.

I suspect 1 is most likely, but the log message and other changes are to help rule out other possibilities, if that crash ever happens again.